README.md

```diff
@@ -30,7 +30,193 @@ Attributes
     <td>log directory</td>
     <td><tt>/var/log/pgpool</tt></td>
   </tr>
+<tr>
```

Comment:
Is it possible to switch this over to a markdown table instead? It'll be a lot easier to read and review.